app/models/setting/discovered.rb

```diff
@@ -42,6 +48,18 @@ def self.load_defaults
       end
     end
 
+    Setting.transaction do
+      [
+          self.set('discovery_CATEGORY_highlights', N_("The default regex to organize facts for highlights section"), "" ),
+          self.set('discovery_CATEGORY_storage', N_("The default regex to organize facts for storage section"), "" ),
+          self.set('discovery_CATEGORY_software', N_("The default regex to organize facts for software section"), "" ),
+          self.set('discovery_CATEGORY_hardware', N_("The default regex to organize facts for hardware section"), "" ),
+          self.set('discovery_CATEGORY_network', N_("The default regex to organize facts for network section"), "" ),
+          self.set('discovery_CATEGORY_ipmi', N_("The default regex to organize facts for ipmi section"), "" ),
+      ].compact.each { |s| self.create s.update(:category => "Setting::Discovered")}
+    end
+
+
```

Comment:
I apologize for complications, but I like `discovery_facts_highlights` much better than (capitalized) CATEGORY.